content/docs/create-a-new-react-app.md

```diff
@@ -1,94 +1,94 @@
 ---
 id: create-a-new-react-app
-title: Create a New React App
+title: 建立全新的 React 應用程式
 permalink: docs/create-a-new-react-app.html
 redirect_from:
   - "docs/add-react-to-a-new-app.html"
 prev: add-react-to-a-website.html
 next: cdn-links.html
 ---
 
-Use an integrated toolchain for the best user and developer experience.
+使用綜合的 toolchain 以得到最佳的使用者和開發者體驗。
 
-This page describes a few popular React toolchains which help with tasks like:
+本頁將介紹一些受歡迎的 React toolchain，它們有助於以下的工作：
 
-* Scaling to many files and components.
-* Using third-party libraries from npm.
-* Detecting common mistakes early.
-* Live-editing CSS and JS in development.
-* Optimizing the output for production.
+* 擴大文件和 component 的規模。
+* 使用來至 npm 的第三方函式庫。
+* 偵測早期常見的錯誤。
+* 實時在開發環境裡編輯 CSS 和 JS。
```

Comment:
Maybe `Live-editing` keep the original term?

Reply:
I think translating `live-editing` wouldn't distort the meaning. Unlike verbs like `rendering` which might cause some technical misunderstanding after translation.

Reply:
Got it!